package.json

```diff
@@ -15,12 +15,15 @@
   "author": "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)",
   "contributors": [
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)",
-    "Christian Murphy <christian.murphy.42@gmail.com> (https://github.com/ChristianMurphy)"
+    "Christian Murphy <christian.murphy.42@gmail.com> (https://github.com/ChristianMurphy)",
+    "Thomas 'zemnmez' Shadwell (https://github.com/zemnmez)"
   ],
   "files": [
     "index.js"
   ],
   "dependencies": {
+    "typescript": "^3.7.4",
+    "unist-util-is": "^4.0.1",
```

Comment:
Could/should these be devDependencies?

Reply:
they should. Tried to add them with `--dev` (like `yarn`) guess it doesnt work the same...

Reply:
yarn uses `--dev`, npm uses `--save-dev`